index.html

```diff
@@ -8679,7 +8662,18 @@
 							   <input type="checkbox" id="adventure_context_mod" style="margin:0px 0 0;">
 							</div>
 							<div id="instructsection" class="settinglabel hidden" style="padding-top: 3px;">
-
+								<!-- TODO: Allow selecting name and picture for yourself and AI. This could be embeded on the left of each message.
+								<table class="settingsmall text-center" style="border-spacing: 4px 2px;	border-collapse: separate;">
+									<tr>
+									  <th>You <a class="color_blueurl" href="#" onclick="selectYourPicture()">🎨</a></th>
+									  <th>AI <a class="color_blueurl" href="#" onclick="selectBotPicture()">🎨</a> <span class="helpicon">?<span class="helptext">The name of the person you want to chat with. Multiple opponents can be specified, creating a group chat, separate their names with ||$||</span></span></th>
+									</tr>
```

Comment:
i'm assuming this is WIP as the function doesn't exist. However if you do plan to add user-uploadable images, i would recommend they be stored as minified base64 strings within the embedded settings on the story JSON - you can refer to the `compressImage` function which I use to pack incoming 512x512 stable horde images into low-res 256x256 base64 encoded data. Ideally this data should not exceed 10kb.

Reply:
I was thinking they'd just select a path which would be used to load the image each time it was needed.
Let me know your thoughts on this or whether you'd rather them be stored as portable images in the JSON instead!

Reply:
Hmm maybe let's skip the user avatar loading first, once we have a baseline for this mode, then you can add them in later.